.github/workflows/branch-build.yml

```diff
@@ -7,8 +7,7 @@ on:
 
```

Comment:
Could you please provide a link to a successful run for the branch build on your fork? I couldn't find any. Generally, after applying changes, it would be great to test PR + branch build once again on the fork, to make sure it'll work after merge. Thanks!

Reply:
https://github.com/ezodude/botkube/commit/34682f851e43c3ae9cac1ddb3e2b5b985f3329f5

Reply:
You can also see this in the screenshot in the PR description.